pragma-node/src/handlers/entries/get_onchain/ohlc.rs

```diff
@@ -146,6 +149,29 @@ impl ChannelHandler<SubscriptionState, SubscriptionRequest, InfraError> for WsOH
 
         match serde_json::to_string(&ohlc_data) {
             Ok(json_response) => {
+                let ip_addr = subscriber.ip_address;
+
+                // Close the connection if rate limit is exceeded.
+                if subscriber.rate_limiter.check_key_n(
```

Comment:
So it checks if this response only isn't exceeding the rate limit?

what happens if the same ip queries 100 times the endpoint, this would not close the connection as the 100 individuals queries are below the limit?
just curious

Reply:
no it checks that with the given message it won't exceed the rate limit which is set as a quote of bytes per second